src/update_roster.R

```diff
@@ -61,8 +55,7 @@ build_rosters <-
     }
 
     if (season >= 2016) {
-      weekly_rosters <- purrr::possibly(build_rosters_weekly_ngsapi,
-                                        data.frame())(season)
+      weekly_rosters <- purrr::possibly(build_rosters_weekly_ngsapi, data.frame())(season)
       if (nrow(weekly_rosters) > 0) {
         roster <- convert_weekly_to_season_rosters(weekly_rosters)
         shield <- build_rosters_shieldapi(season)
```

Comment:
Actually not related to this PR but isn't this line redundant? It's almost the same as line 24 and probably runs every time (because weekly rosters usually have more than 0 lines)

Reply:
Good catch, yeah, I'll do a little rearranging so it only runs once